docs/workforce-management-framework/topics/con-wfm-intergrations.adoc

```diff
@@ -0,0 +1,26 @@
+= {Feature} intergration
+
+An intergration is the use of existing opensource projects to solve specific needs for {Feature}. 
```

Comment:
>existing opensource projects

Im guessing that we are not limited to opensource only?

Reply:
At the moment all open source and ideally to continue that way.